app/src/main/java/com/example/jasmiensofiecels/wordly/model/services/OxfordDictionaryService.java

```diff
@@ -0,0 +1,28 @@
+package com.example.jasmiensofiecels.wordly.model.services;
+
+import com.example.jasmiensofiecels.wordly.presenter.base.dailyWord.DailyWordService;
+
+
+/**
+ * The purpose of this class is to represent the business logic of calling the Oxford Dictionary API to get the word of the
+ * day.
+ * Created by Jasmien Cels on 23/03/2018.
+ */
+
+public class OxfordDictionaryService implements DailyWordService {
+
+
+    public OxfordDictionaryService() {
+
+    }
+
+    //Word object
+
+
+    //TODO: Get the API result object data structure.
+    @Override
+    public void getWordOfTheDay(ResponseListener listener) {
+
+
+    }
+}
```

Comment:
Really like your use of contracting here. That's the dream. Have you considered the overall architecture of your app (perhaps in diagram form)? 

With the use of these contracts I would assume so